CHANGES.rst

```diff
@@ -1,10 +1,11 @@
 Changes
 =======
 
-Version 0.8.2 (UNRELEASED)
+Version 0.9.0 (UNRELEASED)
```

Comment:
Please keep the 0.8.2 one. We can actually commit this fix to `maint-0.8` branch rather than `master`. Since this is a bug fix, it is best to fix it at the last broken version. And it will get propagated to `master` via maint-to-master merge.